cdap-ui/app/cdap/components/ConfigurationGroup/PropertyRow/index.tsx

```diff
@@ -139,24 +161,37 @@ class PropertyRowView extends React.Component<IPropertyRowProps, IState> {
       };
     }
 
+    // When there is only one error and it does not have element property,
+    // it is a property level error.
```

Comment:
You mean a plugin level error? If it does not have a element property does it mean its a plugin level error?

Reply:
It will be widget/property level error. For example in any multi row widgets , if we have 'element', it means the error is in one of the rows of the widget. If we don't have 'element' property, the error is at the widget level.